tests/files/environments/development/us-east-1.yml

```diff
@@ -0,0 +1,15 @@
+defaults:
+  parameters:
+    CidrBlock: '10.0'
+stacks:
+  - name: Test-Dev-VPC
+    template_name: VPC
+    parameters:
+      Name: "TestVPC"
+
+  - name: Test-Dev-SecurityGroup
+    template_name: SecurityGroup
+    parameters:
+      VpcId:
+        Stack: Test-VPC
+        Output: VpcId
```

Comment:
@jfhamlin if you think there's a viable path from this to the future along with some sort of deprecation, I'm all for this.

What about at least starting with just `environments/<env>.yml`, which would contain stacks in any region for that environment for now. You set the default environment at the top, then override in the stack definition if needed. That way we can incrementally move to stack-per-file a little easier?